_data/entertainment.yml

```diff
@@ -104,6 +110,8 @@ websites:
     tfa:
       - totp
     doc: https://qa.nicovideo.jp/faq/show/4500
+    regions:
+      - jp
```

Comment:
```suggestion
```
While the site is oriented to serve customers mainly from Japan, they do [take international payment](http://blog.nicovideo.jp/en_info/2019/11/post-068984.html) and are accessible outside of Japan.